deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -153,6 +153,8 @@
 
     private boolean overrideTimestamps;
 
+    private int filesIngested = 0;
```

Comment:
could you rename this to something like `filesIngestInWork` and/or add a comment explaining that it is tracking the file count within one work, not across the whole deposit? Just seems worth clarifying since its a class level variable